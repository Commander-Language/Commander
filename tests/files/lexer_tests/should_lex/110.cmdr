//Test that type variables lex correctly
type a = (string, string);
type b = a[];
type c = b;
type d = (a, b, c)[];