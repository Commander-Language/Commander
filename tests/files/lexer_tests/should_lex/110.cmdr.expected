TYPE 'type' 2:1
VARIABLE 'a' 2:6
EQUALS '=' 2:8
LPAREN '(' 2:10
STRING 'string' 2:11
COMMA ',' 2:17
STRING 'string' 2:19
RPAREN ')' 2:25
SEMICOLON ';' 2:26
TYPE 'type' 3:1
VARIABLE 'b' 3:6
EQUALS '=' 3:8
VARIABLE 'a' 3:10
LSQUARE '[' 3:11
RSQUARE ']' 3:12
SEMICOLON ';' 3:13
TYPE 'type' 4:1
VARIABLE 'c' 4:6
EQUALS '=' 4:8
VARIABLE 'b' 4:10
SEMICOLON ';' 4:11
TYPE 'type' 5:1
VARIABLE 'd' 5:6
EQUALS '=' 5:8
LPAREN '(' 5:10
VARIABLE 'a' 5:11
COMMA ',' 5:12
VARIABLE 'b' 5:14
COMMA ',' 5:15
VARIABLE 'c' 5:17
RPAREN ')' 5:18
LSQUARE '[' 5:19
RSQUARE ']' 5:20
SEMICOLON ';' 5:21
END_OF_FILE 5:22
